lib/__tests__/findUndefinedIdentifiers-test.js

```diff
@@ -164,6 +164,22 @@ it('knows about objects', () => {
   ).toEqual(new Set(['foo', 'uuid']));
 });
 
+it('knows about identifiers declared after usage', () => {
+  expect(
+    findUndefinedIdentifiers(
+      parse(
+        `
+    function modifyAbc() {
+      abc.def = 4;
+    }
+
+    export const abc = {};
+  `,
```

Comment:
@lencioni I noticed that `prettier` made the indentation of these strings odd. I followed local formatting, but I'd like to hear your thoughts, perhaps it's a bug in prettier?